de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/views/product_specifications_v.sql

```diff
@@ -17,7 +17,7 @@ CREATE OR REPLACE VIEW "de.metas.fresh".product_specifications_v AS
     nf.Name                                                 as nutritionName,
     pn.SeqNo                                                as NutritionSeqNo,
     pn.nutritionqty,
-    coalesce(bomProduct.CustomerLabelName, bomProduct.Name) as componentName,
+    coalesce(NULLIF(bomProduct.CustomerLabelName,''), bomProduct.Name) as componentName,
```

Comment:
```suggestion
    coalesce(NULLIF(TRIM(bomProduct.CustomerLabelName),''), bomProduct.Name) as componentName,
```

..also guard against e.g. `' '`